src/@utils/provider.ts

```diff
@@ -85,16 +87,44 @@ export async function getFileDidInfo(
 
 export async function getFileUrlInfo(
   url: string,
```

Comment:
same here, we add more than URL checking in here so parameter should not be named `url`.

Reply:
this has not been solved. Again, `url` is now only one way of storage. So we can't name everything `url`. Use e.g. `file` here and use `url` naming only for the actual thing, URLs.